src/native-bindings.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const bindings = require(path.join(__dirname, '../', 'build', 'Release', 'exokit.node'));
+const bindings = require(path.join(__dirname, '..', 'build', 'Release', 'exokit.node'));
```

Comment:
Why this change? Does it do anything?

Reply:
It doesn't do anything technically, it makes it less confusing without the slash.